src/JustSaying/AwsTools/MessageHandling/ISqsQueue.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Threading.Tasks;
+using Amazon;
+using Amazon.SQS;
+using JustSaying.AwsTools.QueueCreation;
+
+namespace JustSaying.AwsTools.MessageHandling
+{
+    public interface ISqsQueue
+    {
+        IAmazonSQS Client { get; }
```

Comment:
What problem is this trying to solve? Have you just extracted this out of some of the existing classes?

Reply:
I added the interface so I could mock out the implementation for the SqsNotificationListener tests, previously it had SqsQueueBase in the constructor